base/sparse/cholmod.jl

```diff
@@ -87,22 +87,22 @@ function __init__()
             tmp = Array(Cint, 3)
             ccall((:cholmod_version, :libcholmod), Cint, (Ptr{Cint},), version_array)
             ccall((:jl_cholmod_version, :libsuitesparse_wrapper), Cint, (Ptr{Cint},), tmp)
-            if tmp != version_array
+            if tmp[1] != version_array[1]
```

Comment:
Doesn't this verify ony `x` (not `x` and `y`) in a `x.y.z` version number as @andreasnoack suggested in https://github.com/JuliaLang/julia/pull/10362#issuecomment-135881222. 

I don't know what policy COLMOD has for breaking changes, but I'd recommend verifying before loosening the requirement too much.